content/200-orm/050-overview/500-databases/200-database-drivers.mdx

```diff
@@ -70,3 +70,63 @@ To use this feature:
    - [Neon](/orm/overview/databases/neon#how-to-use-neons-serverless-driver-with-prisma-preview)
    - [PlanetScale](/orm/overview/databases/planetscale#how-to-use-the-planetscale-serverless-driver-with-prisma-preview)
    - [Turso](/orm/overview/databases/turso#how-to-connect-and-query-a-turso-database)
+
+### Driver adapters and custom output paths
+
+Since Prisma 5.9.0, when using the driver adapters Preview feature along with a [custom output path for Prisma Client](/orm/prisma-client/setup-and-configuration/generating-prisma-client#using-a-custom-output-path), you cannot reference Prisma Client using a relative path.
+
+Let's assume you had `output` in your Prisma schema set to `../src/generated/client`:
+
+```prisma
+generator client {
+  provider = "prisma-client-js"
+  output   = "../src/generated/client"
+}
+```
+
+What you should **not** do is reference that path relatively:
+
+```ts no-copy
+// what not to do!
+import { PrismaClient } from './src/generated/client'
+
+const client = new PrismaClient()
+```
+
+Instead, you will need to use a linked dependency.
+
+<TabbedContent tabs={[<FileWithIcon text="npm" icon="file"/>, <FileWithIcon text="pnpm" icon="file"/>, <FileWithIcon text="yarn" icon="file"/>]}>
+
+<tab>
+
+```terminal
+npm add db@./src/generated/client
```

Comment:
What does this really do? Does this add this path as an alias `db`?

Reply:
Yes, this adds the generated client as a dependency with the alias `db`

Reply:
I think that would be good to add to the description. I assume I am not alone in not having used that before, hence not being super clear what it does.